mojaloop/values.yaml

```diff
@@ -6001,6 +6001,637 @@ central:
       # requests:
       #  cpu: 100m
       #  memory: 128Mi
+  emailnotifier:
+    replicaCount: 1
+    image:
+      repository: mojaloop/email-notifier
+      tag: v0.1
```

Comment:
This should match the sprint release.

I believe we are in 4.1.0?

@elnyry please confirm confirm.